security/src/main/java/zone/cogni/lib/security/common/PermissionGlobalMethodSecurityConfiguration.java

```diff
@@ -32,4 +34,11 @@ protected AccessDecisionManager accessDecisionManager() {
     permissionService.ifPresent(service -> manager.getDecisionVoters().add(new PermissionAccessDecisionVoter(service)));
     return manager;
   }
+
+  @ConfigurationProperties(prefix = "cognizone.security")
```

Comment:
@domivds naming convention is not bad: company / module / ... but then I am missing maybe a bit more context. I guess there can also be other things in `cognizone.security`. 

so why not make it `cognizone.security.global-properties`?

Reply:
global = default = root?
but future-wise having it in a sub property will be cleaner :-)